vector/src/main/res/layout/activity_tchap_room_creation.xml

```diff
@@ -36,136 +35,132 @@
+                    android:layout_marginStart="20dp"
                     android:layout_marginLeft="20dp"
-                    android:layout_marginRight="10dp"
                     android:layout_marginTop="15dp"
+                    android:layout_marginEnd="10dp"
+                    android:layout_marginRight="10dp"
                     android:textColorHint="?attr/vctr_default_text_hint_color">
 
                     <com.google.android.material.textfield.TextInputEditText
                         android:id="@+id/et_room_name"
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
                         android:layout_weight="1"
-                        android:textSize="18.8sp"
                         android:hint="@string/tchap_room_creation_name"
                         android:minHeight="40dp"
-                        android:textColor="@color/vector_tchap_text_color_dark_grey"/>
+                        android:textColor="@color/vector_tchap_text_color_dark_grey"
+                        android:textSize="18.8sp" />
                 </com.google.android.material.textfield.TextInputLayout>
             </LinearLayout>
 
             <LinearLayout
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:descendantFocusability="beforeDescendants"
-                android:focusableInTouchMode="true"
+                android:layout_height="match_parent"
+                android:layout_marginStart="10dp"
                 android:layout_marginLeft="10dp"
+                android:layout_marginEnd="7dp"
                 android:layout_marginRight="7dp"
-                android:orientation="horizontal">
+                android:descendantFocusability="beforeDescendants"
+                android:focusableInTouchMode="true"
+                android:gravity="center_vertical"
+                android:minHeight="48dp"
+                android:orientation="vertical">
 
                 <TextView
-                    android:id="@+id/tv_external_access"
-                    android:layout_width="0dp"
-                    android:layout_height="wrap_content"
-                    android:layout_weight="1"
-                    android:text="@string/tchap_room_creation_unrestricted"
-                    android:textSize="15sp"
-                    android:paddingBottom="3dp"
-                    android:textColor="@color/vector_tchap_text_color_dark"/>
-
-                <Switch
-                    android:id="@+id/switch_external_access_room"
+                    android:id="@+id/tv_room_msg_retention"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:checked="false"/>
+                    android:paddingBottom="3dp"
+                    android:text="@plurals/tchap_room_creation_retention"
```

Comment:
This does not work in the preview, you can replace by
`tools:text="@string/tchap_room_settings_retention_title"`

Reply:
Not sure to understand what you mean
The current preview is expected one:
<img width="479" alt="Screenshot 2019-12-09 at 16 52 58" src="https://user-images.githubusercontent.com/8969772/70450685-6d5d2100-1aa4-11ea-8d61-06644d63b0c5.png">